### DISCUSSION TITLE
Documentation with mor information about Instalation and setup

### DISCUSSION THREAD

**Discussion by michelvalent** [opendistro-for-elasticsearch/sql#978] [Ideas]
Hi,

first all, congratulation for the project and thanks for it.

I felt missed in documentation the instruction to install and setup. 

Best regards

**Comment by dai-chen** [comment]
Hi @michelvalent are you looking for ODFE setup doc? https://opendistro.github.io/for-elasticsearch-docs/docs/install/

**Reply by michelvalent** [reply]
Yes, it is.
Thanks Chen.
